ui/src/pages/ShoeMap.tsx

```diff
@@ -101,33 +111,10 @@ export const ShoeMap: React.FC = () => {
           <TileLayer
             url={`https://api.mapbox.com/styles/v1/hanlinc27/ckhjy5wat2dvz1aplv4tkaghb/tiles/{z}/{x}/{y}?access_token=${process.env.REACT_APP_MAPBOX_ACCESS_TOKEN}`}
           />
-          {(() => {
-
-            if(stories && filteredStories.length != 0){
-              var filteredStoryArr= new Array<Story>();
-
-              stories.forEach(function(story){
-                filteredStories.forEach(function (country){
-                  if(story.author_country == country){
-                    filteredStoryArr.push(story);
-                  }
-                });
-              });
-    
-              return (
-              <PinCluster stories={filteredStoryArr} openDrawer={handleOpenDrawer} />
-              )
-            } 
-            
-            else if(stories){
-              return(
-                <PinCluster stories={stories} openDrawer={handleOpenDrawer} />
-              )
-            }
-
-          })()}
+          {stories && !error && (
+            <PinCluster stories={stories} openDrawer={handleOpenDrawer} />
```

Comment:
Can we create a Github issue to render stuff in error states?

Will require some design work as well.

Reply:
Done!